RxJavaTemplate/app/src/main/java/co/nimblehq/rxjava/ui/screens/home/HomeFragment.kt

```diff
@@ -89,5 +90,7 @@ class HomeFragment : BaseFragment<FragmentHomeBinding>() {
     private fun showLoading(isLoading: IsLoading) {
         binding.btHomeRefresh.isEnabled = !isLoading
         viewLoadingBinding.pbLoading.visibleOrGone(isLoading)
+
+        IdlingResource.increaseOrDecrease(isLoading)
```

Comment:
Should we mark this `VisibleForTesting`?

Reply:
Please correct me if I'm wrong but I'm not quite sure this is necessary 🙏🏼  Could you please tell me where exactly you think we should add this annotation?

Reply:
@chornerman About `@VisibleForTesting`, please refer [this](https://developer.android.com/reference/kotlin/androidx/annotation/VisibleForTesting)

One more thing, I see you created `IdlingResource` on `staging` and `production`, it's quite redundant IMO. 
Here is the other approach from Android official [testing-sample](https://github.com/android/testing-samples/blob/456eadca14a5cd64e8787deaa4e4cec25ca27587/ui/espresso/IdlingResourceSample/app/src/main/java/com/example/android/testing/espresso/IdlingResourceSample/MainActivity.java#L44)
We can define `IdlingResource` as nullable and skip it to leave it as null in production code.

Reply:
Use Thread.sleep() instead 🙏🏼